ignite/pom.xml

```diff
@@ -36,25 +36,33 @@ LICENSE file.
     <skipJDK9Tests>true</skipJDK9Tests>
     <skipJDK10Tests>true</skipJDK10Tests>
     <skipJDK11Tests>true</skipJDK11Tests>
+    <gridgain.version>8.8.34</gridgain.version>
   </properties>
 
+  <repositories>
+    <repository>
+      <id>gridgain-external-repository</id>
+      <url>https://www.gridgainsystems.com/nexus/content/repositories/external</url>
+    </repository>
+  </repositories>
+
   <dependencies>
     <dependency>
-      <groupId>org.apache.ignite</groupId>
+      <groupId>org.gridgain</groupId>
       <artifactId>ignite-core</artifactId>
-      <version>${ignite.version}</version>
+      <version>${gridgain.version}</version>
```

Comment:
Should not we use `<scope>provided</scope>` here for all `org.gridgain` artifacts?

Reply:
Probably. I'm not sure. `ignite3-binding` module hasn't `provided` scope for AI3 as well.

Reply:
Updated and tested with `provided` scope. Works well with [poc-tester branch qa-4256](https://github.com/ggprivate/poc-tester/tree/qa-4256).